detector/fcal/BeamCal_o1_v01_geo.cpp

```diff
@@ -45,6 +50,10 @@ static DD4hep::Geometry::Ref_t create_detector(DD4hep::Geometry::LCDD& lcdd,
   const double bcalInnerR = dimensions.inner_r();
   const double bcalOuterR = dimensions.outer_r();
   const double bcalInnerZ = dimensions.inner_z();
+  if (bcalInnerZ/dd4hep::mm > 3200) {
```

Comment:
About checking the end of BeamCal: You are perfectly right. The agreement with MDI refers to the beginning, but the check should be made at the end for a number of reasons.
Question: checking the end will make the dimension outer_z mandatory in the BeamCal xml file. This will need to be synchronised in all xml files using this driver. Is this ok?